lib/service.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/**
+ * Dependencies
+ */
+
+var format = require('util').format;
+var objectAssign = require('object-assign');
+
+var pullImage = require('./pull-image');
+var run = require('../util/run');
+
+
+/**
+ * Services
+ */
+
+var services = {
+	mongodb: {
+		host: 'db',
```

Comment:
The hostname that should be used in the code to connect with mongodb. It is the alias in the `--link` flag.

```
mongo.connect('mongodb://db/test');
```